kubernetes/base/config/modules.yaml

```diff
@@ -10,13 +10,13 @@ modules:
   mod_privacy: {}
   mod_vcard:
     search: true
-  ModBeamStats: {}
-  mod_pottymouth:
-    blacklists:
-      default: /home/ejabberd/.ejabberd-modules/sources/mod_pottymouth/blacklist/blacklist_en.txt
-      en: /home/ejabberd/.ejabberd-modules/sources/mod_pottymouth/blacklist/blacklist_en.txt
-  ModPushSkillz:
-    rabbit_url: "amqp://RABBITMQ_AUTH@rabbit.qa.skillz.com:5672/skillz" # Populated by a secret macro
+  # ModBeamStats: {}
```

Comment:
Pretty sure we still want these modules

Reply:
Currently disabled in base so ejabberd can be started on its own ( without rabbitmq or mysql ). I can re-add modbeamstats I think.